### ISSUE TITLE
Unsupported Asset KP3RBNB

### ISSUE THREAD

**Issue by lucaswalter** [JKorf/Binance.Net#491]
**Describe the bug**
Currently running into an issue where we are iterating over every product pair from Binance and get an `ArgumentException` for the asset pair: KP3RBNB

Checking on Binance, and it appears this is a valid pair: https://www.binance.com/en/trade/KP3R_BUSD

**To Reproduce**
Called using the `GetMyTradesAsync` endpoint:
```
                        var tradesResult = await client.Spot.Order.GetMyTradesAsync(
                            symbol: "KP3RBNB",
                            limit: MAX_PAGE_SIZE,
                            fromId: tradeId,
                            receiveWindow: RECEIVE_WINDOW);
```

**Expected behavior**
An ArgumentException is not thrown.

**Comment by glres** [comment]
#489

**Comment by JKorf** [comment]
I've merged your PR, should be fixed in version 6.3.3.

**Comment by glres** [comment]
Thanks
